underlay/js/bigipreport.js

```diff
@@ -2070,9 +2161,10 @@ function showPreferences(updatehash) {
     //Populate the content
     $("div#preferences").html(settingsContent);
 
-    //Populate the settings according to the local storage or default settings of none exist
+    //Populate the settings according to the local storage or default settings if none exist
     $("#autoExpandPools").prop("checked", localStorage.getItem("autoExpandPools") === "true");
     $("#adcLinks").prop("checked", localStorage.getItem("showAdcLinks") === "true");
+    $("#regexSearch").prop("checked", localStorage.getItem("regexSearch") === "true");
```

Comment:
This checkbox is missing in my version (I think)?

Reply:
It's on the preferences page on mine. Label:
"Use Regular Expressions when searching"
Now defaulting to on.